src/coredump/coredumpctl.c

```diff
@@ -915,14 +986,18 @@ static int save_core(sd_journal *j, FILE *file, char **path, bool *unlink_temp)
         /* Look for a coredump on disk first. */
         r = sd_journal_get_data(j, "COREDUMP_FILENAME", (const void**) &data, &len);
         if (r == 0) {
+                char *resolved = NULL;
+
                 r = retrieve(data, len, "COREDUMP_FILENAME", &filename);
                 if (r < 0)
                         return r;
                 assert(r > 0);
 
-                if (access(filename, R_OK) < 0)
+                if (chase_symlinks_and_access(filename, arg_root, CHASE_PREFIX_ROOT, F_OK, &resolved, NULL) < 0)
```

Comment:
`chase_symlinks_and_access()` does not return its error via `errno`, so this needs to be changed to:

```
r = chase_symlinks_and_access(filename, arg_root, CHASE_PREFIX_ROOT, F_OK, &resolved, NULL);
if (r < 0)
        ...
```